From c6197e093c71fa2fc1136be33ef6e7a6a827e5b9 Mon Sep 17 00:00:00 2001
From: Pawit Pornkitprasan <p.pawit@gmail.com>
Date: Sat, 24 Jan 2015 12:43:38 +0700
Subject: [PATCH] Revert "Update Larger icons setting for better usability"

Not working properly on hdpi. Revert while waiting for fix.

This reverts commit 5c7529006f3ed78d934fed2861cebd3654695108.
---
 src/com/android/launcher3/DynamicGrid.java | 38 +++++++++++++++---------------
 1 file changed, 19 insertions(+), 19 deletions(-)

diff --git a/src/com/android/launcher3/DynamicGrid.java b/src/com/android/launcher3/DynamicGrid.java
index 1494311..d2a5ff7 100644
--- a/src/com/android/launcher3/DynamicGrid.java
+++ b/src/com/android/launcher3/DynamicGrid.java
@@ -34,7 +34,7 @@ public class DynamicGrid {
     private float mMinHeight;
 
     // This is a static that we use for the default icon size on a 4/5-inch phone
-    static float DEFAULT_ICON_SIZE_DP = 66;
+    static float DEFAULT_ICON_SIZE_DP = 60;
     static float DEFAULT_ICON_SIZE_PX = 0;
 
     public static float dpiFromPx(int size, DisplayMetrics metrics){
@@ -67,47 +67,47 @@ public class DynamicGrid {
         DEFAULT_ICON_SIZE_PX = pxFromDp(DEFAULT_ICON_SIZE_DP, dm);
         // Our phone profiles include the bar sizes in each orientation
         deviceProfiles.add(new DeviceProfile("Super Short Stubby",
-                255, 300,  2, 3,  (useLargeIcons ? 58 : 46), 13, (hasAA ? 3 : 5),
-                (useLargeIcons ? 58 : 46), fourByFourDefaultLayout,
+                255, 300,  2, 3,  (useLargeIcons ? 54 : 48), 13, (hasAA ? 3 : 5),
+                (useLargeIcons ? 54 : 48), fourByFourDefaultLayout,
                 R.xml.default_workspace_4x4_no_all_apps));
         deviceProfiles.add(new DeviceProfile("Shorter Stubby",
-                255, 400,  3, 3,  (useLargeIcons ? 58 : 46), 13, (hasAA ? 3 : 5),
-                (useLargeIcons ? 58 : 46), fourByFourDefaultLayout,
+                255, 400,  3, 3,  (useLargeIcons ? 54 : 48), 13, (hasAA ? 3 : 5),
+                (useLargeIcons ? 54 : 48), fourByFourDefaultLayout,
                 R.xml.default_workspace_4x4_no_all_apps));
         deviceProfiles.add(new DeviceProfile("Short Stubby",
-                275, 420,  3, 4,  (useLargeIcons ? 58 : 46), 13, (hasAA ? 5 : 5),
-                (useLargeIcons ? 58 : 46), fourByFourDefaultLayout,
+                275, 420,  3, 4,  (useLargeIcons ? 54 : 48), 13, (hasAA ? 5 : 5),
+                (useLargeIcons ? 54 : 48), fourByFourDefaultLayout,
                 R.xml.default_workspace_4x4_no_all_apps));
         deviceProfiles.add(new DeviceProfile("Stubby",
-                255, 450,  3, 4,  (useLargeIcons ? 58 : 46), 13, (hasAA ? 5 : 5),
-                (useLargeIcons ? 58 : 46), fourByFourDefaultLayout,
+                255, 450,  3, 4,  (useLargeIcons ? 54 : 48), 13, (hasAA ? 5 : 5),
+                (useLargeIcons ? 54 : 48), fourByFourDefaultLayout,
                 R.xml.default_workspace_4x4_no_all_apps));
         deviceProfiles.add(new DeviceProfile("Nexus S",
-                296, 491.33f,  4, 4,  (useLargeIcons ? 58 : 46), 13, (hasAA ? 5 : 5),
-                (useLargeIcons ? 58 : 46), fourByFourDefaultLayout,
+                296, 491.33f,  4, 4,  (useLargeIcons ? 54 : 48), 13, (hasAA ? 5 : 5),
+                (useLargeIcons ? 54 : 48), fourByFourDefaultLayout,
                 R.xml.default_workspace_4x4_no_all_apps));
         deviceProfiles.add(new DeviceProfile("Nexus 4",
-                335, 567,  4, 4,  (useLargeIcons ? DEFAULT_ICON_SIZE_DP : 52), 13, (hasAA ? 5 : 5),
-                (useLargeIcons ? 60 : 46), fourByFourDefaultLayout,
+                335, 567,  4, 4,  (useLargeIcons ? DEFAULT_ICON_SIZE_DP : 56), 13, (hasAA ? 5 : 5),
+                (useLargeIcons ? 56 : 48), fourByFourDefaultLayout,
                 R.xml.default_workspace_4x4_no_all_apps));
         deviceProfiles.add(new DeviceProfile("Nexus 5",
-                359, 567,  4, 4,  (useLargeIcons ? DEFAULT_ICON_SIZE_DP : 52), 13, (hasAA ? 5 : 5),
-                (useLargeIcons ? 60 : 46), fourByFourDefaultLayout,
+                359, 567,  4, 4,  (useLargeIcons ? DEFAULT_ICON_SIZE_DP : 56), 13, (hasAA ? 5 : 5),
+                (useLargeIcons ? 56 : 48), fourByFourDefaultLayout,
                 R.xml.default_workspace_4x4_no_all_apps));
         deviceProfiles.add(new DeviceProfile("Large Phone",
-                406, 694,  5, 5,  (useLargeIcons ? 68 : 52), 14.4f,  5, (useLargeIcons ? 60 : 44),
+                406, 694,  5, 5,  (useLargeIcons ? 64 : 56), 14.4f,  5, (useLargeIcons ? 56 : 48),
                 R.xml.default_workspace_5x5, R.xml.default_workspace_5x5_no_all_apps));
         // The tablet profile is odd in that the landscape orientation
         // also includes the nav bar on the side
         deviceProfiles.add(new DeviceProfile("Nexus 7",
-                575, 904,  5, 6, (useLargeIcons ? 76 : 60), 14.4f,  7, (useLargeIcons ? 64 : 52),
+                575, 904,  5, 6, (useLargeIcons ? 72 : 60), 14.4f,  7, (useLargeIcons ? 60 : 52),
                 R.xml.default_workspace_5x6, R.xml.default_workspace_5x6_no_all_apps));
         // Larger tablet profiles always have system bars on the top & bottom
         deviceProfiles.add(new DeviceProfile("Nexus 10",
-                727, 1207,  5, 6,  (useLargeIcons ? 80 : 64), 14.4f,  7, (useLargeIcons ? 68 : 56),
+                727, 1207,  5, 6,  (useLargeIcons ? 76 : 64), 14.4f,  7, (useLargeIcons ? 64 : 56),
                 R.xml.default_workspace_5x6, R.xml.default_workspace_5x6_no_all_apps));
         deviceProfiles.add(new DeviceProfile("20-inch Tablet",
-                1527, 2527,  7, 7,  (useLargeIcons ? 104 : 80), 20,  7, (useLargeIcons ? 76 : 64),
+                1527, 2527,  7, 7,  (useLargeIcons ? 100 : 80), 20,  7, (useLargeIcons ? 72 : 64),
                 fourByFourDefaultLayout, R.xml.default_workspace_4x4_no_all_apps));
         mMinWidth = dpiFromPx(minWidthPx, dm);
         mMinHeight = dpiFromPx(minHeightPx, dm);
-- 
1.9.3 (Apple Git-50)

